src/Sulu/Bundle/ResourceBundle/Resource/Exception/FilterContextNotFoundException.php

```diff
@@ -27,10 +27,10 @@ class FilterContextNotFoundException extends FilterException
      */
     private $name;
 
-    public function __construct($id)
```

Comment:
is this exception for a missing "filter" feature in a context? maybe you could rename it to `MissingFeatureException` then it is reusable for upcoming features!